invenio_records_ui/permissions.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
```

Comment:
`None` looks fine because of this [if](https://github.com/inveniosoftware/invenio-records-ui/blob/4c4770d37960abe4656693625944003279972fa9/invenio_records_ui/views.py#L186)